e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs

```diff
@@ -137,5 +143,20 @@ public void Dispose()
 
         string IOpcPlcConfig.Urls => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PLC_SIMULATION_URLS,
             () => throw new Exception("Semicolon separated list of URLs of OPC-PLCs is not provided."));
+
+        string IContainerRegistryConfig.DockerServer => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_DOCKER_SERVER,
+            () => throw new Exception("Server name is not provided"));
+
+        string IContainerRegistryConfig.DockerUser => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_DOCKER_USER,
+            () => throw new Exception("Username is not provided"));
+
+        string IContainerRegistryConfig.DockerPassword => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_DOCKER_PASSWORD,
+            () => throw new Exception("Password is not provided"));
+
+        string IContainerRegistryConfig.ImagesNamespace => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_IMAGES_NAMESPACE,
+            () => throw new Exception("Namespace is not provided"));
+
+        string IContainerRegistryConfig.ImagesTag => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_IMAGES_TAG,
+            () => Assembly.GetExecutingAssembly().GetReleaseVersion().ToString(3));
```

Comment:
> Assembly.GetExecutingAssembly().GetReleaseVersion().ToString(3)) [](start = 17, length = 65)

Value have to be provided by DevOps pipeline, we should not support fallbacks here, please throw exception

Reply:
changed